frontend/app/teams/[team]/page.jsx

```diff
@@ -0,0 +1,21 @@
+import { findTeam } from '../../../airtable/utils';
+import Admin from '../../../components/Admin/Admin';
```

Comment:
Next gives you the ability to create absolute file imports. Im pretty sure this could become:
```
'components/Admin/Admin';
```
but if not, there is an easy to configure an alias for imports. They are cleaner and more reliable in the chance you were to start moving files around. 

https://nextjs.org/docs/advanced-features/module-path-aliases